chart/templates/_helpers.tpl

```diff
@@ -322,6 +322,16 @@ Create the name of the service account to use for the workspace-api deployment
 {{- end -}}
 {{- end -}}
 
+{{/*
+If we connect to an Azure Postgres database, the internal user and the user used to connect to the database are different
```

Comment:
I would explain this a little bit better, and even putting an example on how they look. Similar as you wrote in the readme